br/pkg/restore/import.go

```diff
@@ -366,10 +367,12 @@ func (importer *FileImporter) Import(
 					logutil.ShortError(errDownload))
 				return errors.Trace(errDownload)
 			}
-			log.Debug("download file done", zap.String("file-sample", files[0].Name), zap.Stringer("take", time.Since(start)),
+			log.Info("download file done", zap.String("file-sample", files[0].Name), zap.Stringer("take", time.Since(start)),
 				logutil.Key("start", files[0].StartKey),
 				logutil.Key("end", files[0].EndKey),
+				logutil.Region(info.Region),
 			)
+			downloadSucceed = true
```

Comment:
Seems that any successful region will make `downloadSucceed` be true, but we expect it to be "all successful" ?